liferay-echo/liferay-portal
# LPS-179883  Copy permissions on the new page when a Page is created with Copy Page

[Link to the story](https://issues.liferay.com/browse/LPS-177031)

Motivation
=======
Copy permissions on the new page when a Page is created with Copy Page

Proposed Solution
========

- Extract copy page to the api with copy permissions 
- Add new dropdown item contextual to use it in the item list of actions.

Steps to Verify:
========

1. Open the Site Builder
2. Create a new Blanck Page called Page 1
3. Publish Page 1
4. Click on One Page Options Menu -> Permissions
5. Remove all the permissions for Guest
6. Click on One Page Options Menu -> Copy Page with Permissions
7. Name new page as Page 2
8. Verify that new permissions are similar to the copied page

Notes
========
Integration test included.
